src/optical_flow.jl

```diff
@@ -102,25 +102,36 @@ a default value of four is assumed.
 The algorithm calculates the minimum eigenvalue of a (2 x 2) normal matrix of
 optical flow equations, divided by number of pixels in a window; if this value
 is less than `eigenvalue_threshold`, then a corresponding feature is filtered
-out and its flow is not processed (default value is 10^-6).
+out and its flow is not processed (default value is 1e-4).
+
+## `ϵ` termination criteria
+
+Minimum required change in displacement at which the iterative algorithm continues to work.
+Default value is `1e-2`.
 
 ## References
 
 1. B. D. Lucas, & Kanade. "An Interative Image Registration Technique with an Application to Stereo Vision," DARPA Image Understanding Workshop, pp 121-130, 1981.
 2. J.-Y. Bouguet, “Pyramidal implementation of the afﬁne lucas-kanade feature tracker description of the algorithm,” Intel Corporation, vol. 5,no. 1-10, p. 4, 2001.
 """
 struct LucasKanade{F <: Float64, I <: Int}  <: OpticalFlowAlgorithm
-	iterations::I
+    iterations::I
     window_size::I
     pyramid_levels::I
     eigenvalue_threshold::F
+    ϵ::Float64
 end
```

Comment:
No parameters actually.
```julia
struct LucasKanade <: OpticalFlowAlgorithm
    iterations::Int
    window_size::Int
    pyramid_levels::Int
    eigenvalue_threshold::Float64
    ϵ::Float64
end
```

Reply:
Sorry, didn't get what you mean here. Do you suggest removing this parameter?

Reply:
Nevermind, got it :)